### ISSUE TITLE
SmartCity-Accessibility Server Side #447 failed

### ISSUE THREAD

**Issue by JuanJenkinsBot** [TechnionYP5777/SmartCity-Accessibility#290]
Build 'SmartCity-Accessibility Server Side' is failing!

Failed Tests :
```
No tests ran.
```

Changes since last successful build:
- [KoralChapnik] 86456e83164875c2f195727499c1f06925ca4bc4 - updated bestReviewsTest #228

- [KoralChapnik] 1961786c3fbcc9e6c54bb5fb69e3b2d626084a61 - forgot to remove the ignore in bestReviewsTest #228

- [KoralChapnik] 5a2e34c6a5bd677aeccf1a4d40994c784f408c00 - change best reviews to consider admin reviews with negative helpfilness

- [KoralChapnik] 27ea24d7ba311190a5f0eff20d7ccd40ce01115c - added Helpfulness test #228

- [yaelAmitay] 17e2765c295144847f0be1a1434f7fff42f730b2 - Navigation get token from header #277

- [yaelAmitay] 3cd2d4edcfc2d605e99e8fae92929e49cfc4555c - minor changes to login

- [yaelAmitay] 62a239f3afa4d571fb8db3b102228fb2676a2fe4 - login and navigation lot of code #252 and #277 and #276

- [yaelAmitay] bae9f69d70bcaf81fdc7e422a37243d707e8f925 - fix token in navigation clientside #276

- [yaelAmitay] 768572979fe2ad8f9e5c743fec96ffc0124cf5a7 - navigation server side #277

- [yaelAmitay] fd98faa01d61dcdd9b59d31ab89e680facdb563b - allow global cors #287

- [Arthur Sapozhnikov] 4552dd95f9967ea4c8c7f2d7d3d11ba5d4d45a31 - Working on making a normal and functional review page based on Yael's

- [yaelAmitay] 17e60407343022af474cd647a43747b674e22c7e - add constants file #288

- [yaelAmitay] 92025be546ae4a673fc5e08d063aa95e06adb10b - add AddReviewService to providers

[View full output](http://kapalex.ddns.net:8080/job/SmartCity-Accessibility%20Server%20Side/447/)

**Comment by yaelAmitay** [comment]
Why does it suddenly warn about it?

**Comment by kapalex** [comment]
the previous build was stuck in an infinte loop (please set timeouts on tests) so I manually closed it and started a new build

**Comment by yaelAmitay** [comment]
my tests have timeout since last semester.
Does anyone have any idea what the bug is?

**Comment by koralchapnik** [comment]
How does we know where the infinite loop is ?

**Comment by kapalex** [comment]
You can see [here](http://kapalex.ddns.net:8080/job/SmartCity-Accessibility%20Server%20Side/446/console) where it got stuck
seems like somewhere in searchtests

**Comment by JuanJenkinsBot** [comment]
Build was fixed!
